lti_consumer/lti_1p3/ags.py

```diff
@@ -37,6 +38,8 @@ def __init__(
         # Lineitems urls
         self.lineitems_url = lineitems_url
 
+        self.lineitem_url = lineitem_url
+
```

Comment:
@shimulch Looks like I've removed the code in a previous PR.
Can you modify `get_lti_ags_launch_claim` below in the file and add the `lineitem_url` claim if it's set?
The final claim should look like this: https://www.imsglobal.org/spec/lti-ags/v2p0#example-link-with-one-line-item-tool-has-all-permissions

Reply:
@giovannicimolin, Updated.